docs/html/index.html

```diff
@@ -1,23 +1,23 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>Welcome to nginx!</title>
+<title>Welcome to NGINX!</title>
```

Comment:
This is a wrong change: nginx is written in lowercase as "nginx".

Also, it is out of scope of this PR, because this has no relation to updating links as stated in PR subject.